packages/cli/src/templates/spec.tpl

```diff
@@ -0,0 +1,43 @@
+import { TestBed } from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { NgxsModule, Select, Store } from '@ngxs/store';
+import { Observable } from 'rxjs';
+import { {{pascalCase name}}State, {{pascalCase name}}StateModel } from './{{dashCase name}}.state';
+import { {{pascalCase name}}Action } from './{{dashCase name}}.actions';
+
+describe('{{pascalCase name}} actions', () => {
+  const states = [{{pascalCase name}}State];
+
+  @Component({ selector: 'test-component', template: '' })
+  class TestComponent {
+    @Select({{pascalCase name}}State) {{camelCase name}}State$: Observable<{{pascalCase name}}StateModel>;
+    constructor(private store: Store) { }
+    addItem() {
+      this.store.dispatch(new {{pascalCase name}}Action('item-1'));
+    }
+  }
```

Comment:
I think that it would be better to test the state class through the store directly and not via a component because component tests are quite expensive and are actually adding no real value here other than as a container for the store.

This spec file should be an example of how we recommend testing the state. Look here for inspiration:
https://ngxs.gitbook.io/ngxs/recipes/unit-testing

Reply:
done